generation/day_ahead.py

```diff
@@ -114,6 +114,12 @@ def import_day_ahead_generation(
     abort_if_data_empty(scheduled_generation)
     log.debug("Overall aggregated generation: \n%s" % scheduled_generation)
 
+    log.debug("Up-sampling overall aggregated generation ...")
+    if pd.infer_freq(scheduled_generation.index) != "15T":
```

Comment:
Just to be clear: We're hard-coding the resolution here. Didn't we just move it from within code into the sensor config?